### PR TITLE
Update to GnuPG version 2.2.21

### PR DESCRIPTION
**Pull request by cwross**

### CODE REVIEW

**Comment by cwross** [comment]
Version 2.2.21 appears to resolve the issues reported in #5193

**Comment by extrowerk** [comment]
Please remove the old recipe.
Thanks!

File: app-crypt/gnupg/gnupg-2.2.21.recipe [hunk 1]
```diff
@@ -105,6 +104,8 @@ TEST_REQUIRES="
 	cmd:ssh
 	"
 
+PATCHES="gnupg-2.2.21.patchset"
```
Comment (extrowerk) [inline app-crypt/gnupg/gnupg-2.2.21.recipe:107]: Please move this back to the line after the CHECKSUM (eg. into line 20)

Comment (extrowerk) [reply app-crypt/gnupg/gnupg-2.2.21.recipe:107]: And please use the $portVersion variable instead of hardcoding the version number here.

Comment (cwross) [reply app-crypt/gnupg/gnupg-2.2.21.recipe]: Requested changes made.

**Comment by extrowerk** [comment]
closes #5193

**Pull request merged**